test/network/BancorNetwork.ts

```diff
@@ -2129,22 +2129,42 @@ describe('BancorNetwork', () => {
 
             value ||= sourceTokenAddress === NATIVE_TOKEN_ADDRESS ? amount : BigNumber.from(0);
 
-            return network
-                .connect(trader)
-                .tradeBySourceAmount(
-                    sourceTokenAddress,
-                    targetTokenAddress,
-                    amount,
-                    minReturnAmount,
-                    deadline,
-                    beneficiary,
-                    {
-                        value
-                    }
-                );
+            let retVal = BigNumber.from(0);
+            if (getRetVal) {
+                retVal = await network
+                    .connect(trader)
+                    .callStatic.tradeBySourceAmount(
+                        sourceTokenAddress,
```

Comment:
You can group these arguments in an external array and pass is as `...args` in both of the calls.